index.html

```diff
@@ -10,125 +10,143 @@
+            <h2>Tonic</h2>
+            <ul class="tech">
+              <li class="company-name"><a>company</a></li>
+              <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+              <li><a>Back End Dev</a></li>
+              <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+              <li><a>2015</a></li>
+            </ul>
+            <P>A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+            <ul class="prog-languages">
+              <li class="lang">html</li>
+              <li class="lang">css</li>
+              <li class="lang">javaScript</li>
+            </ul>
+            <button type='button'>See Project</button>
+          </div>
         </section>
 
         <section class="project">
-          <img src="./images/Snapshoot-Portfolio4.png" alt="snapshot1">
-          <h2>Multi-Post Stories</h2>
-          <ul class="tech">
-            <li class="company-name"><a>company</a></li>
-            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
-            <li><a>Back End Dev</a></li>
-            <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
-            <li><a>2015</a></li>
-          </ul>
-          <P>A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
-          <ul class="prog-languages">
-            <li class="lang">html</li>
-            <li class="lang">css</li>
-            <li class="lang">javaScript</li>
-          </ul>
-          <button type='button'>See Project</button>
+          <img class="snapshot2" src="./images/Snapshoot-Portfolio4.png" alt="snapshot1">
+          <div class="project-properties">
+            <h2>Multi-Post Stories</h2>
+            <ul class="tech">
+              <li class="company-name"><a>company</a></li>
+              <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+              <li><a>Back End Dev</a></li>
+              <li><a href="#"><img src="images/Counter.png" alt="dot"></a></li>
+              <li><a>2015</a></li>
+            </ul>
+            <P>A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+            <ul class="prog-languages">
+              <li class="lang">html</li>
+              <li class="lang">css</li>
+              <li class="lang">javaScript</li>
+            </ul>
+            <button type='button'>See Project</button>
+          </div>
         </section>
       </section>
 
       <section id="about-me">
-        <div><h2>About</h2><h2>Myself</h2></div>
-        <p>Hello I’m a software developer! I can help you build a product,
-           feature or website Look through some of my work and experience! 
-           If you like what you see and have a project you need coded, 
-           don’t hestiate to contact me.
-        </p>
+        <div id="right-side">
+          <div><h2>About</h2><h2>Myself</h2></div>
```

Comment:
- [ Optional] The font-size of the about me section in the mobile version seems quite big in relation to the others (you are using 1.5em font-size here). Would you please check it.
![about-me-title-font-size](https://user-images.githubusercontent.com/44057863/166106072-2e643b85-c916-41c4-bb0a-41a684617365.png)